src/main/java/io/appium/java_client/pagefactory/README.md

```diff
@@ -0,0 +1,160 @@
+# Standard Selectors
```

Comment:
Please move this document to `docs` folder. There should be no documents inside java sources

Reply:
I can do that.